src/player.cpp

```diff
@@ -14442,40 +14442,76 @@ std::vector<mission*> player::get_failed_missions() const
     return failed_missions;
 }
 
-void player::print_encumbrance(WINDOW *win, int min, int max, int line) const
+encumbrance_data player::get_encumbrance( size_t i ) const
+{
+    encumbrance_data e;
```

Comment:
It's semi-ok to have single character variables in very short lambdas and loops, especially when the variable is an anonymous piece of data, but give a meaningful name whenever possible, for example 'i' should be something like body_part_index, and e should reference what it's doing (accumulating encumbrance data).